### ISSUE TITLE
[BUG] Extension messes up on refresh

### ISSUE THREAD

**Issue by chky-nmnsoftware** [pixijs/devtools#39]
When refreshing, the extension should "refresh" as well to accommodate any changes that were made. Instead, the extension remains after a refresh, which cases it to no longer have the correct references. This might be caused due to the way Chrome Extensions work, though the ultimate solution would be for the extension to refresh alongside the page as well.

**Comment by Zyie** [comment]
Hey, when using the devtool are you using `@pixi/devtool`, writing your own `window.__PIXI_DEVTOOLS__ = {...}` or using the automatic detection from pixi?

**Comment by chky-nmnsoftware** [comment]
> Hey, when using the devtool are you using `@pixi/devtool`, writing your own `window.__PIXI_DEVTOOLS__ = {...}` or using the automatic detection from pixi?

I am writing my own using the following:
```
globalThis.__PIXI_DEVTOOLS__ = {
    pixi: PIXI,
    app: this.game.app,
    extensions: [
        . . .
    ]
}
```
(Of course, the `extensions` array has actual content)

**Comment by Zyie** [comment]
Hey @chky-nmnsoftware can you check if this has been fixed? the `2.0.3` version is the latest

**Comment by chky-nmnsoftware** [comment]
> Hey @chky-nmnsoftware can you check if this has been fixed? the `2.0.3` version is the latest

Currently on version `2.0.3`. I reinstalled the plugin and even downloaded the release from GitHub and loaded the unpacked version of it. Unfortunately the bug is still present. From what it looks like, it does "refresh" after I reload the page, but then it goes back to its previous state (when reloading, it goes from the tree to the first screen that shows how to use the extension, and then back to the tree, but exactly how it was before the reload [which is holding wrong references]).

**Comment by chky-nmnsoftware** [comment]
@Zyie I'm closing this issue since it seems like it works correctly in the newest version. Thank you for the help!
